### ISSUE TITLE
Throttle command doesn't execute on the trailing end

### ISSUE THREAD

**Issue by martinstuecklschwaiger** [you-dont-need/You-Dont-Need-Lodash-Underscore#286]
`_.throttle` rightfully executes both on the leading and trailing edge of the timeout. 

```JS
f = _.throttle(() => console.log("x"), 500);
f();
f();
f();

// --- will output ---

// x
// x ... after 500ms
```

The function in the repository only executes on the leading edge of the timeout, so the later invocation won't happen at all. This is actually needed and useful if you want to throttle resize and scroll events.

**Comment by flying-sheep** [comment]
yes, basically if the third option is passed to throttle, this lint shouldn’t trigger
